ehitchhiking-ui/src/app/shared/components/trip/trip.component.html

```diff
@@ -23,7 +23,9 @@
 				star
 			</i>
 		</button>
-
+    <button class="trip-btn" (click)="openRatePassengersDialog()"><i class="material-icons">
```

Comment:
This icon shouldn't be visible for a  trip performed as passenger. Pls extend RatePassegnersModal.

- Add possibility to change header title to 'Rate Driver'
- Rename 'exit trip' button to 'Submit'
- Add control to close modal to the header as it's done for other modals

Reply:
fixed